Person.py

```diff
@@ -0,0 +1,43 @@
+from random import SystemRandom
+from Room import Office, LivingSpace
+
+
+class Person:
```

Comment:
make this an abstract class. This is to make sure that you cannot create an instance of person. But rather a fellow or staff.